server/server.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+const port = process.env.PORT || 5000;
+const bodyParser = require("body-parser");
+require("dotenv").config();
```

Comment:
If we got time we can put in axios later

Reply:
i back-end side does not care about fetch or axios :)